src/main/java/org/opentripplanner/graph_builder/GraphBuilder.java

```diff
@@ -159,13 +162,18 @@ public static GraphBuilder create(CommandLineParameters params, GraphConfig conf
                         LOG.info("Skipping DEM file {}", file);
                     }
                     break;
+                case NETEX_NO:
```

Comment:
On further reflection, and seeing other comments on Netex implementation, I realize there is only ever intended to be one Netex importer, which may gradually be expanded to handle the EU profile instead of Norway. Unless there are plans to somehow detect whether a zipped Netex package contains EU or Norway Netex, the enum value should just be `NETEX` with no suffix.